### ISSUE TITLE
Inference script

### ISSUE THREAD

**Issue by HeChengHui** [yuanc3/Pose2ID#3]
@yuanc3 
Thank you for your work! It is an interesting concept and i would like to use with my reid model. 
Is it possible for you to provide a sample script on how to use your work? 
Thank you!

**Comment by HeChengHui** [comment]
@yuanc3 

Assuming i am trying to do ReID on a video would the steps be? :

pre-process
1. run IPG on the gallery to get other poses
2. for each unique ID, run reid model to get features of all its poses and run the code in quickstart (IPG part)

main
1. compare detected person reid features with the features computed in pre-process

**Comment by yuanc3** [comment]
pre-process
1. run IPG on the gallery or gallery or both (depends on the real scene) to get other poses.
2. for each ***samlple*** and its generated other poses, run reid model to get features of all its poses 

* The code in quickstart (IPG part) is just a demo, it depends on the test way of your codes. All in all, the main idea is to add features together and normalize them.

In short, reid is computing distances between every query sample (image) and gallery sample (image), you can use IPG on each sample (image) to centralize its feature for a better retrieval.

**Comment by HeChengHui** [comment]
some questions:
1. Must the gallery images be front facing? or does it work with people captured at odd angles.
2. Must the gallery images be close up shots?
3. Do i have to run nfc on the normalised fused features?
```
feats = torch.nn.functional.normalize(feats + eta * feats_ipg, dim=1, p=2) # L2 normalization
feats = NFC(feats, k1 = 2, k2 = 2)
```

**Comment by HeChengHui** [comment]
Thanks for your quick reply!
